organization/_posts/2018-05-16-packaging.md

```diff
@@ -0,0 +1,62 @@
+---
+title: "HSF Packaging Group Meeting #24, May 16, 2018"
+layout: default
+---
+
+# {{page.title}}
+
+Agenda
+======
+[https://indico.cern.ch/event/727088/](https://indico.cern.ch/event/727088/)
+
+Participants: Graeme Stewart, Ben Morgan, Oana Boeriu, Patrick Gartung, Chris Burr, Rafal Pacholek, Javier Cervantes Villanueva, Guilherme Amadio, Chris Green
+
+[Introduction - Graeme](https://indico.cern.ch/event/727088/contributions/2992555/attachments/1643141/2640438/HSF_Packaging_Group_Intro_2018-05-16.pdf)
+===========================
+- Follow on from micro-architecture builds: Ben has [link on HSF/packaging github](https://github.com/HSF/packaging/tree/master/istools) to discuss this/collate information. PRs to update welcome!
+- [HSF tech note on naming build platforms](http://hepsoftwarefoundation.org/technical_notes.html) promoted to consultation document
+  - Needs people to take a look and finalise
+  - E.g., Add micro-microarchitecture piece, compiler release numbers (e.g. for GCC only major version should matter now)
+  - How to comment/edit? To check, but probably on HSF/technotes GitHub.
+- Graeme: Overview on [Guix Cern Computing Seminar](https://indico.cern.ch/event/719851/) (pronounced "gix"!)
+  - No slides, but link to recording available
+  - Nutshell: very similar to nix in how packages built, hashed.
+  - High reproducibility (up to 95% bitwise identical).
+  - Nix-style runtime env setup
+  - Everything lives under `/guix` (must be writable, user builds supported by daemon). Binary builds supported, but not relocatable (prefix enters package hash).
+  - User Base: mostly HPC, big shared filesystems user are close to (Graeme commented at seminar that typical HEP case is different; CVMFS etc)
+  - Good points like Nix: reproducibility, user control, composability (overlays)
+  - Differences: Scheme language vs Nix DSL (claimed Scheme makes use/extension easier). No mention of license
+  - Interesting for us? Yes, but lives very close to Nix. Limited resources, so not a priority unless someone wants to take it on. Maybe more important if we move in a Nix-like direction. Then differences between Nix DSL vs Scheme etc.
```

Comment:
Slightly odd last sentence. Better would be "Then differences between Nix DSL vs Scheme, for example, would be more relevant to investigate."